tom_nonlocalizedevents/alertstream_handlers/igwn_event_handler.py

```diff
@@ -87,7 +87,7 @@ def handle_igwn_message(message: JSONBlob, metadata: Metadata):
                 )
                 logger.error(traceback.format_exc())
 
-    logger.debug(f"Storing igwn alert: {alert}")
+    logger.info(f"Storing igwn alert: {alert}")
```

Comment:
@griffin-h by making this an INFO message (vs. a DEBUG) message it'll added to the log when the logger debug level is INFO, which it normally is. So _more_ often. As written, the alert is only logged when the logging level is DEBUG, which should only be the case during development and not in production.

So, I'm thinking you don't want make this (L#90) change. Unless, I'm missing someting?